tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 3.0
-envlist = py3{7,8,9,10}
```

Comment:
@benkiel, does it make sense to specify this at all? Looking at the output of test runs, I see a lot of messages like "py3x: skipped because could not find python interpreter with spec(s): py3x" in each. This seems to duplicate what you specify in GitHub workflows config, so I wonder if there's any sense to keep this line (and maintain it in sync with what you have in `run-tests.yaml`) at all.